php/utils-wp.php

```diff
@@ -319,11 +319,29 @@ function wp_get_table_names( $args, $assoc_args = array() ) {
 	global $wpdb;
 
 	$tables = array();
+
```

Comment:
We need to prepend a check and throw an error if people try to use both `--base-tables-only` and `--views-only` at the same time.

Reply:
I've added the check/error, but no test coverage for this as I wasn't sire if it merited one?